contribute_to_pim/translate.rst

```diff
@@ -5,31 +5,60 @@ We use `Crowdin`_ to translate the PIM.
 
 .. image:: ./crowdin-logo.png
 
-How to translate
-----------------
+How to translate?
+-----------------
 
-Don't hesitate to create an account and begin to translate on: https://crowdin.com/project/akeneo
+You have to to create an account and begin to translate in your favorite language on `Crowdin`_.
+You need a proofreading permission to validate the strings, so your translation will be included in the PIM.
+Each language with more than 80% of valid translations are automatically translated with `Nelson`_ each week, and will be available in the next patch.
+You will be able to claim your badge "El Translator" on `Badger platform <http://badger.akeneo.com/login/>`_!
 
-If your language is not enabled, don't hesitate to ask, we'll enable it with pleasure.
+Your language is not available on Crowdin?
+------------------------------------------
 
-If you want to become a proofreader and be able to validate translations for a language, don't hesitate to ask.
+As Akeneo PIM is Open Source, we will be glad to add welcome new languages for our application.
+Don't hesitate to ask to a `Crowdin`_ administrator to add your language to the available languages list of Crowdin.
+Please reminder that only languages validated in more than 80% will be synchronized in Akeneo PIM.
 
-Don't hesitate to claim your badge "El Translator" on `Badger platform <http://badger.akeneo.com/login/>`_!
+You want to use a non fully translated language in your application?
+--------------------------------------------------------------------
 
-How it works
-------------
+.. note::
 
-When a new translation key is added in the code of the PIM, this key is sent to Crowdin.
+    We strongly encourage you to use the default way, that is validate the application in more than 80% and wait for the next patch.
+    Indeed, an application not fully translated will contains a lot of non translated strings in many places of your application and can result of a very bad user experience.
 
-Keys are translated by contributors, then validated by proof readers.
+By default, Akeneo PIM only displays and synchronize languages validated in more than 80%.
```

Comment:
```suggestion
By default, Akeneo PIM only displays and synchronizes languages validated in more than 80%.
```
Also, " in more than 80%" seems weird, any English reviewer to suggest a better thing?